programs/zstdcli.c

```diff
@@ -720,6 +730,10 @@ int main(int argCount, const char* argv[])
         BMK_setBlockSize(blockSize);
         BMK_setNbThreads(nbThreads);
         BMK_setNbSeconds(bench_nbSeconds);
+        BMK_setLdmFlag(ldmFlag);
+        BMK_setLdmMinMatch(g_ldmMinMatch);
+        BMK_setLdmHashLog(g_ldmHashLog);
+        BMK_setLdmHashEveryLog(g_ldmHashEveryLog);
```

Comment:
Add `if (g_ldmHashEveryLog != LDM_HASHEVERYLOG_DEFAULT)` so we don't depend on `LDM_HASHEVERYLOG_DEFAULT` being the same across modules.